tests/manage/z_cluster/test_custom_scc.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.utility import templating
+from ocs_ci.helpers.helpers import create_resource
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.pod import (
+    Pod,
+    get_pods_having_label,
+    wait_for_pods_to_be_running,
+)
+from ocs_ci.ocs.exceptions import TimeoutExpiredError
+from ocs_ci.framework.pytest_customization.marks import bugzilla, polarion_id, tier2
+from ocs_ci.helpers.sanity_helpers import Sanity
+
+logger = logging.getLogger(__name__)
+
+
+class TestSCC:
+    @pytest.fixture(scope="function")
+    def create_scc(self, request):
```

Comment:
Why not throw this in helpers so it would be reusable by someone else? And also create a fixture in conftest so the deletion will be automatic in test teardown. The first I think is mandatory, the second I leave it for you to decide.

Reply:
makes sense!